src/deps_info.json

```diff
@@ -94,5 +94,11 @@
   "_embind_register_std_string": ["malloc", "free"],
   "_embind_register_std_wstring": ["malloc", "free"],
   "__syscall192": ["emscripten_builtin_memalign"],
-  "pthread_create": ["malloc", "free", "emscripten_main_thread_process_queued_calls"]
+  "pthread_create": ["malloc", "free", "emscripten_main_thread_process_queued_calls"],
+  "$getTypeName": ["free"],
```

Comment:
is this one needed? I think deps_info is for C symbols only - the linker that reads that file does not see the JS anyhow. So it must map a C symbol it sees to something that symbol will cause a JS library to require later.

Reply:
Right - removed.